docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+
+  app:
+    image: ghcr.io/radu-c-martin/maybe:latest
+    ports:
+      - 3000:3000
+    restart: unless-stopped
+    env_file:
+      - .env
+    environment:
+      RUN_DB_MIGRATIONS_IN_BUILD_STEP: "false"
+      DB_HOST: "postgres"
+      HOSTING_PLATFORM: "localhost"
+      DISABLE_SSL: "true"
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+
+  redis:
```

Comment:
One question, why do we need redis here?

Reply:
I based the docker-compose on the devcontainer one, which has redis as well, but as far as I can see it's not actually used?

Reply:
We won't need Redis for this first pass.  It is not being used within the app currently.